### PR TITLE
REL-3145: ITAC emails must be adjusted to explain time allocation breakdown

### PR DESCRIPTION
**Pull request by cquiroz**
This fixes differences between the OT and ITAC on the way the program and partner time are calculated

### CODE REVIEW

File: phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/TimeAmount.java [hunk 1]
```diff
@@ -18,6 +18,8 @@
     protected TimeUnit units;
     private static final double EQUALS_TOLERANCE = 0.00001d;
 
+    private static final double HOURS_TO_MILLIS = 60 * 60 * 10000;
```
Comment (sraaphorst) [inline phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/TimeAmount.java]: Maybe I'm brain dead today, but shouldn't this be
`60 * 60 * 1000` (and not `10000`)?

Comment (swalker2m) [inline phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/TimeAmount.java]: Or maybe `Duration.ofHours(1).toMillis()`

Comment (sraaphorst) [inline phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/TimeAmount.java]: Yes, that's definitely a better way.

Comment (cquiroz) [inline phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/TimeAmount.java]: Doh, Will do the `Duration` version. This maybe unnecessary as the conversion back and forth cancels the constant but i'm worry about the original ITAC implementation of the conversion so i did my own

**Pull request merged**